### PR TITLE
Move to reactstrap/bootstrap for UI.

### PR DESCRIPTION
**Pull request by akdombrowski**
screenshot_2018-12-25 react app
screen shot 2018-12-25 at 15 59 00
screen shot 2018-12-25 at 15 59 09
screen shot 2018-12-25 at 15 59 14
screen shot 2018-12-25 at 15 59 17

### CODE REVIEW

File: client/src/components/GameArea.js [hunk 1]
```diff
@@ -43,30 +61,286 @@ class GameArea extends Component {
...
+            "max-height": "100vh"
+          }
+        }
+      >
+
+        <Row
+          className="top-nav-row p-0 m-0"
+          style={
+            {
+              // "max-height": "9vh",
```
Comment (jtenberg) [inline client/src/components/GameArea.js]: nit: Is this needed anymore?

File: client/src/components/GameArea.js [hunk 2]
```diff
@@ -43,30 +61,286 @@ class GameArea extends Component {
...
+        <div className="mh-100">
+          <h6 className="display-5 mh-100">Main Board</h6>
+          <p className="d-none d-sm-flex">View of your battlefield.</p>
         </div>
-      </div>
+      </Jumbotron>
+    );
+  }
+
+  lifeComponent() {
```
Comment (jtenberg) [inline client/src/components/GameArea.js]: We should eventually break this out into its own component.

File: client/src/components/GameArea.js [hunk 3]
```diff
@@ -43,30 +61,286 @@ class GameArea extends Component {
...
+    })
+  }
+
+  decrement(x) {
+    this.setState({
+      life: Number(this.state.life) - Number(x)
+    })
+  }
+
+  addCardToTopBattlefield(name, cost, img, type, set, text, power, toughness) {
```
Comment (jtenberg) [inline client/src/components/GameArea.js]: Nothing to change but will this method eventually be sending actions / updating state?

File: client/src/styles/Card.css [hunk 4]
```diff
@@ -1,95 +1,7 @@
-.card-parent-container {
-  height: 100vh;
-  max-height: 100vh;
-  padding-right: 56.25%;
+/* p {
```
Comment (jtenberg) [inline client/src/styles/Card.css]: Do we need this file or is it a placeholder for future css?

Comment (jtenberg) [reply client/src/styles/Card.css]: and similarly for other css files?

File: client/src/components/Card.js [hunk 5]
```diff
@@ -1,48 +1,207 @@
...
+              "max-height": "15%"
+            }
+          }
+        >
+          {/* Card name */}
+          <Col
+            className="card-name-col px-0 flex-shrink-1 flex-grow-5 justify-content-start"
+          >
+            <p
+              style={
```
Comment (jtenberg) [inline client/src/components/Card.js]: I'd imagine it is not easy but we may want to consider refactoring these inline styles to some CSS.

File: client/src/styles/Card.css (hunk 4 above)
Comment (akdombrowski) [reply client/src/styles/Card.css]: I wasn't sure how others felt about doing the inline css versus breaking it out into separate css folders. Any preference? @jtenberg

File: client/src/components/GameArea.js (hunk 3 above)
Comment (akdombrowski) [reply client/src/components/GameArea.js]: I was thinking this method just be kept separate for rendering Cards.

File: client/src/components/GameArea.js (hunk 2 above)
Comment (akdombrowski) [reply client/src/components/GameArea.js]: I was going to put buttons in the top nav bar to have a simple life counter for yourself, but we'll have to come up with an idea for how we want to deal with life counts, your own and others' in the game.

Comment (eeeeeric) [reply client/src/components/GameArea.js]: I think on our personal device, we just show our own life total, and on the big screen, we can include everyone's life (and counters and/or commander damage).

File: client/src/components/Card.js [hunk 6]
```diff
@@ -1,48 +1,207 @@
 import React, { Component } from 'react';
 import '../styles/Card.css';
+import {
```
Comment (hrecker) [inline client/src/components/Card.js]: nit: I don't think all of these imports are used here, as far as I can tell (Jumbotron for example). The same might be true for other components

File: client/src/styles/Card.css (hunk 4 above)
Comment (jtenberg) [reply client/src/styles/Card.css]: I think eventually we should separate it out if it becomes too cumbersome. If there isn't a lot of inline stuff, we can keep it the way it is. We can leave it for now but keep that as a consideration.

**Pull request merged**